packages/components/src/components/hds/advanced-table/index.ts

```diff
@@ -559,7 +560,7 @@ export default class HdsAdvancedTable extends Component<HdsAdvancedTableSignatur
 
     if (reorderedMessageText !== undefined) {
       this.reorderedMessageText = reorderedMessageText;
-    } else {
+    } else if (!isDestroyed(this) && !isDestroying(this)) {
```

Comment:
fwiw: I [tried a similar approach a while back](https://github.com/hashicorp/design-system/pull/3195) and it seemed to solve the issue with `hdsIntl` service